src/AllThemeResources.h

```diff
@@ -371,3 +371,7 @@ from there.  Audacity will look for a file called "Pause.png".
    DEFINE_COLOUR( clrSpectro4Sel,         wxColour(  191,    0,    0),  wxT("Spectro4Sel") );
    DEFINE_COLOUR( clrSpectro5Sel,         wxColour(  191,  191,  191),  wxT("Spectro5Sel") );
 
+   DEFINE_COLOUR( clrClipAffordanceOutlinePen,    wxColour(    0,    0,    0),  wxT("ClipAffordanceOutlinePen") );
+   DEFINE_COLOUR( clrClipAffordanceInactiveBrush, wxColour(  202,  202,  202),  wxT("ClipAffordanceUnselectedBrush") );
+   DEFINE_COLOUR( clrClipAffordanceActiveBrush,   wxColour(  219,  219,  219),  wxT("ClipAffordanceSelectedBrush") );
+   DEFINE_COLOUR( clrClipAffordanceStroke,        wxColour(  255,  255,  255),  wxT("ClipAffordanceStroke") );
```

Comment:
Oh no, adding colors to the theme.  That has not been done in a long time -- since 2017.  James Crook is more familiar with the theming system than anyone.  A question is whether these choices of colors need to be different in other themes.  If so, it's not obvious how to do it.  It involved DarkThemeAsCeeCode.h, etc. which are generated by strange processes.

Reply:
I was relying on https://wiki.audacityteam.org/wiki/Theming_HowTo, page doesn't provide information on theme source code generation though

Reply:
I think it involved making a special build with EXPERIMENTAL_THEME_PREFS and doing other things.

But I also think the colors at the end of the table are just used unchanged in the other themes that don't redefine them.

If you had inserted the lines in the middle, I think that might have messed up the non-default themes worse if they were not regenerated.

I think there is much inconvenience in extending the theming system, which needs rewrites.  But that's another topic.